Userland/Libraries/LibWeb/HTML/DedicatedWorkerGlobalScope.idl

```diff
@@ -1,3 +1,4 @@
+#import <HTML/Window.idl>
```

Comment:
Why this import?

Reply:
This is indeed not needed, but why it was introduced in the first place is related to https://github.com/LadybirdBrowser/ladybird/issues/1855